setup.cfg

```diff
@@ -32,13 +32,13 @@ install_requires =
     nibabel >=3.0.1
     nipype >=1.5.1,<2.0
     traits <6.4
-    niworkflows ~= 1.6.0
+    niworkflows @ git+https://github.com/nipreps/niworkflows.git@master
     nitransforms >= 21.0.0
     numpy >= 1.21.0
-    pybids >= 0.12.1
+    pybids >= 0.15.1
```

Comment:
Is this needed or can we hold off until a new minor release

Reply:
Required by niworkflows, actually.